frontend/partners.html

```diff
@@ -131,7 +131,7 @@
            <div class="col s12 m6 l6">
              <div class="card small">
                <div class="card-image">
-                 <img src="images/c4a.png">
+                 <img class="more-hight" src="images/c4a.png">
```

Comment:
What is "hight"?

Reply:
It has been given a more descriptive class name.

Reply:
@andela-angene  We still have "more-hight"?

Reply:
@chegejames updated.